### ISSUE TITLE
provide dynamically loading client settings by object instead of using load_by_string

### ISSUE THREAD

**Issue by yo8568** [googleads/google-ads-python#69]
Following the example, we need to make a yaml file to set client settings and use `GoogleAdsClient.load_from_storage` to create a new client instance, but it does not make sense in production service that provides multiple customers to fetch their ads.

My current solution is like this,
```python
   configuration_string = '\n'.join([
        'developer_token: %s' % developer_token,
        'client_id: %s' % client_id,
        'client_secret: %s' % client_secret,
        'refresh_token: %s' % refresh_token,
        'login_customer_id: %s' % login_customer_id
    ])
    client = google.ads.google_ads.client.GoogleAdsClient.load_from_string(configuration_string)
```
I use `GoogleAdsClient.load_from_string` to make a string as yaml format, but I think it had better provide a function to load settings by dictionary(object), and it will be more easy to use.

**Comment by BenRKarl** [comment]
@yo8568 thank you for submitting this issue. When you refer to enabling multiple customers to fetch ads, are you using different sets of credentials? Or using a single set of credentials to access various different customer accounts?

**Comment by yo8568** [comment]
Hi @BenRKarl , also thanks for reply.

I would let our account managers to manage ads on my system, and each account manager has their own MCC account that was assigned ad accounts in by scope of business. So, we need to use multiple sets of credentials to let each request independent and including access limitation of account.

**Comment by BenRKarl** [comment]
@yo8568 Thanks for that info, it does make sense. I'll keep this issue open as a feature request to add a method `load_from_dict` that allows the client to receive a dict of credentials.

In the meantime you can sort-of achieve this by initializing the client directly. You'll need to import the OAuth client and create a set of Credentials through it, like so (we do this [here](https://github.com/googleads/google-ads-python/blob/master/google/ads/google_ads/client.py#L62) in the client):

```python
from google.oauth2.credentials import Credentials
from google.auth.transport.requests import Request
from google.ads.google_ads.client import GoogleAdsClient
 
credentials = Credentials(
    None,
    refresh_token=refresh_token,
    client_id=client_id,
    client_secret=client_secret,
    token_uri='https://accounts.google.com/o/oauth2/token')

credentials.refresh(Request())

client = GoogleAdsClient(credentials, developer_token, login_customer_id=login_customer_id)
```

You can reference the initialization method [here](https://github.com/googleads/google-ads-python/blob/master/google/ads/google_ads/client.py#L154) to see what other keyword parameters are available.

**Comment by BenRKarl** [comment]
PR to add `load_from_dict` here: https://github.com/googleads/google-ads-python/pull/139

**Comment by BenRKarl** [comment]
#139 was merged and release in library version 3.1.0.
